stl/inc/ranges

```diff
@@ -6255,7 +6255,7 @@ namespace ranges {
             _STL_VERIFY(_Pred, "cannot increment a chunk_by_view iterator whose parent view has no predicate");
 #endif // _ITERATOR_DEBUG_LEVEL != 0
 
-            const auto _Not_pred = [&_Orig_pred = *_Pred]<class _Ty1, class _Ty2>(_Ty1&& _Left, _Ty2&& _Right) {
+            const auto _Not_pred = [&_Orig_pred = *_Pred]<class _Ty1, class _Ty2>(_Ty1&& _Left, _Ty2&& _Right) -> bool {

```

Comment:
I am wondering whether it would be worth it to use a simple struct with a templated call operator to avoid the potential bloat from the various lambda invocations

Reply:
I do not understand why we aren't using `not_fn` here...

Reply:
> I do not understand why we aren't using `not_fn` as the standard suggests here...

I don't want to drag in `<functional>`, and the use of `not_fn(bind(ref(*pred_), _2, _1))` (in the reference implementation of `find-prev` in [p2443](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2021/p2443r1.html#pnum_16)) makes me uneasy. I believe that by using one lambda, instead of three call wrappers, we can avoid a lot of forwarding.